sopel_modules/imdb/imdb.py

```diff
@@ -30,11 +29,19 @@ def configure(config):
         'Enter omdbapi.com API Key:'
     )
 
+def setup(bot):
+    bot.config.define_section('imdb', IMDBSection)    
+    if not bot.memory.contains('url_callbacks'):
+        bot.memory['url_callbacks'] = tools.SopelMemory()
+    bot.memory['url_callbacks'][imdb_re] = imdb_re
+
+def shutdown(bot):
+    del bot.memory['url_callbacks'][imdb_re]
 
-@commands('imdb')
+@commands('imdb', 'movie')
 @example('.imdb ThisTitleDoesNotExist', '[Error] Movie not found!')
-@example('.imdb Citizen Kane', '[Movie] Title: Citizen Kane | Year: 1941 | Rating: 8.4 | Genre: Drama, Mystery | IMDB Link: http://imdb.com/title/tt0033467')
-@example('.imdb Chuck', '[Series] Title: Chuck | Seasons: 5 | Year: 2007–2012 | Rating: 8.2 | Genre: Action, Comedy, Drama | IMDB Link: http://imdb.com/title/tt0934814')
+@example('.imdb Citizen Kane', '[Movie] Title: Citizen Kane | Year: 1941 | Rating: 8.3 | Tomatometer: 100% | Genre: Drama, Mystery | Plot: Following the death of a publishing tycoon, news reporters scramble to discover the meaning of his final utterance. | IMDB Link: http://imdb.com/title/tt0033467')
+@example('.imdb Chuck', '[Series] Title: Chuck | Seasons: 5 | Year: 2007–2012 | Rating: 8.2 | Genre: Action, Comedy, Drama | Plot: When a twenty-something computer geek inadvertently downloads critical government secrets into his brain, the C.I.A. and the N.S.A. assign two age[…] | IMDB Link: http://imdb.com/title/tt0934814')
```

Comment:
Can you add an example in here for a year? Like Halloween (2018) and Halloween (1978)